Gruntfile.js

```diff
@@ -277,6 +287,8 @@ module.exports = function( grunt ) {
 	grunt.loadNpmTasks( 'grunt-wp-readme-to-markdown');
 	grunt.loadNpmTasks( 'grunt-zip' );
 
+	grunt.registerTask( 'update-assets', [ 'copy:select2' ] );
```

Comment:
It does need to be called manually. It will probably happen when we are prepping the release PR (not packaging/deploying it as it will be committed). I'll add it to our docs when we merge.